Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -169,9 +169,10 @@ - (void)setBridge:(RCTBridge *)bridge
 
 RCT_EXPORT_METHOD(removeAnimatedEventFromView:(nonnull NSNumber *)viewTag
                   eventName:(nonnull NSString *)eventName)
+            animatedNodeTag:(nonnull NSNumber *)animatedNodeTag
```

Comment:
Oh wow, so that might explain why it didn't crash when I was testing (probably did most testing on iOS then tested quickly on Android where the crash actually happens).